### DISCUSSION TITLE
[Feature] Streamlined Photo Viewing in Map View

### DISCUSSION THREAD

**Discussion by DanielLaberge** [immich-app/immich#11959] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

After clicking a cluster in the map view, a grid button that shows all the cluster's photos in a grid, like the home page, would be a great improvement in usability. A timeline would be a nice bonus.

Currently, one needs to view the cluster's photos one by one, which is tedious and not conducive to find specific photos taken at one location, or to quickly see an overview of a trip taken somewhere.

![image](https://github.com/user-attachments/assets/3f2829e3-cee4-4795-9b89-8fd4d57a55cf)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4519

**Reply by DanielLaberge** [reply]
Thanks. GitHub search failed me somehow.

**Comment by Stronginthearm42** [comment]
Or just do it like PhotoPrism

![grafik](https://github.com/user-attachments/assets/f5b0d99e-1454-4007-80b2-532b271783d3)
